packages/docs/celo-codebase/protocol/proof-of-stake/becoming-a-validator.md

```diff
@@ -1,7 +1,7 @@
 # Becoming a Validator
 
-To participate in the network, an operator must put up a slashable bonded deposit, register as a validator, and join a validator group. A minimum stake of one Celo Gold and a notice period of 60 days is required to be a validator in the Alfajores Testnet.
+To participate in the network, an operator must put up a slashable commitment of locked Celo Gold, register as a validator, and join a validator group. A minimum stake of one Celo Gold and a notice period of 60 days is required to be a validator in the Alfajores Testnet.
```

Comment:
"locked" here, "Locked" in governance.md above. I guess it is kind of awkward when used in a sentence, especially with the full "Celo Gold" like here.

Reply:
yep! I plan to rewrite + expand this section of the docs in the near future